humility-cmd/src/idol.rs

```diff
@@ -340,6 +340,36 @@ fn call_arg(
                 }
             }
         }
+        HubrisType::Array(&HubrisArray { goff, count }) => {
+            let t = hubris.lookup_type(goff)?;
+            if !matches!(
+                t,
+                HubrisType::Base(HubrisBasetype {
+                    encoding: HubrisEncoding::Unsigned,
+                    size: 1,
+                })
+            ) {
+                bail!(
+                    "array type in {} ({t:?}) not yet supported",
+                    member.name
+                );
+            }
+            match value {
+                IdolArgument::String(value) => {
```

Comment:
Do we ever want to pass raw bytes? I could imagine wanting some way to pass in a hex string.

Reply:
Maybe, but I don't think it's worth doing now.

Doing a quick audit, the only other case where we pass an array as argument is `image_hash`, which doesn't seem like it would be called from the CLI:
```
"fetch_host_phase2_data": (
   doc: "Instruct the control plane agent to fetch a portion of a host phase 2 image, identified by `hash` and starting at `offset`. When the data arrives, the control plane agent will notify the calling task, at which point it should call `get_host_phase2_data`.",
   args: {
       // Hash of the phase 2 image the host wants.
       "image_hash": "[u8; 32]",
       // Offset within the image the host wants.
       "offset": "u64",
       // Notification bit (i.e., log2 of the actual IRQ value) we
       // should use when the data has been fetched.
       "notification_bit": "u8",
```